release-notes/opensearch-asynchronous-search.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,7 @@
+## Version 2.3.0.0 2022-09-13
```

Comment:
https://github.com/opensearch-project/asynchronous-search/tree/2.x/release-notes --- but 2.x has already has 2.3 rel notes right ? any idea why this is coming as new changes

Reply:
I'm also not very sure, this is the local diff:


> git diff main upstream/2.x
>diff --git a/README.md b/README.md
>index 66d7bb4..ac31a48 100644
>--- a/README.md
>+++ b/README.md
>@@ -116,11 +116,6 @@ Additionally, it is possible to attach one debugger to the cluster JVM and anoth
 >
 >
 >
>-## Getting Help
>-
>-If you find a bug, or have a feature request, please don't hesitate to open an issue in this repository.
>-
>-For more information, see the [project website](https://opensearch.org/) and [technical documentation]>(https://opensearch.org/docs/latest/search-plugins/async/index/). If you need help and are unsure where to open an issue, try the OpenSearch [Forum](https://forum.opensearch.org/).
> 
 >## Security


```